<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthForm</class>
 <widget class="QWidget" name="AuthForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>531</width>
     <height>297</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="hello_text">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Добро пожаловать!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="login">
      <property name="text">
       <string>Логин:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="login_label">
      <property name="text">
       <string>Введите логин...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="password">
      <property name="text">
       <string>Пароль:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="password_label">
      <property name="text">
       <string>Введите пароль...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="email">
      <property name="text">
       <string>Адрес электронной почты:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="email_label">
      <property name="text">
       <string>Введите e-mail...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QPushButton" name="sign_in">
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sign_up">
        <property name="text">
         <string>Регистрация</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
